### DISCUSSION TITLE
change font/size for 'line numbers'?

### DISCUSSION THREAD

**Discussion by ecooch** [WinEdt-Team/WinEdt-Support#51] [1.1 General Help and Discussion]
In WinEdt 11.2, is there a way to change the font (or, at least the font size) used for the line numbers? As far as I could tell from the help file, this isn't a 'changeable option', but maybe there is a way that isn't mentioned in the help. Basically, I like having line number, but in other 'editors' I use, I often reduce the size of the font (relative to the size of the text I'm editing) to make them less conspicuous. [The other way I've done this is to change the colour to something fairly 'faint' - visible, but again, not obtrusive.]

Thanks in advance...

**Comment by Alex-Simonic** [comment]
You can customize the color of text used for line numbers. Modify the value:

bscLineNumbers="#848888"

If you use themes, edit the theme through Options -> Themes -> Edit Current Theme. If you are using the Default Theme you can add the following to your Private Settings:

```
[COLORS*]

COLOR_SCHEME="Color Base: Custom"
COLORS="END_LIST"
  bscLineNumbers="#ff0000" //Make it Red! //"506080"
END_LIST
END="Color Base: Custom"
```

You can double-click on the bscLineNumbers and get a GUI to select a new color (if editing RGB is not your cup of tea). It can be done.

**Reply by ecooch** [reply]
Many thanks. Colour, but not font size, it seems? No matter - colour to 'faint grey'  (RGB #ededed) should suffice. 

Again, many thanks.
